spec/spec_helper.rb

```diff
@@ -7,6 +7,8 @@
 require 'mocha/api'
 require 'bourne'
 require 'ostruct'
+require 'pathname'
+require 'activerecord-import'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.